README.md
# dbd-vue

## Project setup
```
yarn install
```

### Compiles and hot-reloads for development
```
yarn serve
```

### Compiles and minifies for production
```
yarn build
```

### Lints and fixes files
```
yarn lint
```

### Customize configuration
See [Configuration Reference](https://cli.vuejs.org/config/).


CHANGELOG.md
# Changelog

All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.

### [0.1.11](https://github.com/joeytierney/dbd-vue/compare/v0.1.10...v0.1.11) (2022-12-23)


### Features

* 404 html page ([b9619e9](https://github.com/joeytierney/dbd-vue/commit/b9619e97155f7b02198c5e016caf34e2112b12d6))
* chapter 26 killer and survivor character portraits added ([41a482c](https://github.com/joeytierney/dbd-vue/commit/41a482c81be74e3e75f91072cdd273d85272f4d4))
* new killer perks from chapter 26 added to KillerPerkArray.js ([01001c4](https://github.com/joeytierney/dbd-vue/commit/01001c4c51a3b0fc6c3e24bf974ab6fc028f3959))
* new killer The Knight added to KillerArray.js ([0c28584](https://github.com/joeytierney/dbd-vue/commit/0c2858484c8ebd4a46bbf045a28fc1dd27c841f9))
* new survivor perks from chapter 26 added to SurvivorPerkArray.js ([1a744c4](https://github.com/joeytierney/dbd-vue/commit/1a744c448fa8924203c3085154d316c40a39a33a))
* new survivor Vittorio Toscano added to SurvivorArray.js ([e6583e0](https://github.com/joeytierney/dbd-vue/commit/e6583e05860ff02b4a332f44a387ded126d2de32))
* perk icons added for both chapter 26 killer and survivor ([0a51c1a](https://github.com/joeytierney/dbd-vue/commit/0a51c1a2c5d13f17df250bb55fcddde28f8184ce))

### [0.1.10](https://github.com/joeytierney/dbd-vue/compare/v0.1.9...v0.1.10) (2022-09-03)


### Features

* HUNK character portrait ([5d92ac1](https://github.com/joeytierney/dbd-vue/commit/5d92ac1d03f95549a92bc01cbb4d5de1b0786c8d))

### [0.1.9](https://github.com/joeytierney/dbd-vue/compare/v0.1.8...v0.1.9) (2022-08-31)


### Features

* HUNK (The Grim Reaper) added to killers ([796f2e0](https://github.com/joeytierney/dbd-vue/commit/796f2e0f0a3400127982d54ac3e4967dae5caf22))
* new killer perks added ([41905dc](https://github.com/joeytierney/dbd-vue/commit/41905dc965db5c4ca17adcb325a8eb4d7fc7db24))
* new survivor perks added ([290a90b](https://github.com/joeytierney/dbd-vue/commit/290a90b1701af95f3cd13dd53ccad3ff5b3ff93c))
* Sheva and Carlos portraits added, Jill's updated ([6a18b44](https://github.com/joeytierney/dbd-vue/commit/6a18b441c41f4b4c70583eeb613ed2b0880deb21))


### Bug Fixes

* social media links not opening in new tab ([a94008f](https://github.com/joeytierney/dbd-vue/commit/a94008f3fd960b5b8bd53a0b651940d6d4ea3fd1))

### [0.1.8](https://github.com/joeytierney/dbd-vue/compare/v0.1.7...v0.1.8) (2022-08-23)


### Features

* 404 page component, content and style ([1db4309](https://github.com/joeytierney/dbd-vue/commit/1db430995f06871507833c72464e24093e8baa37))
* AboutPage component and content created ([c564db6](https://github.com/joeytierney/dbd-vue/commit/c564db61ae98e7b40c27ce2ba06a37420a2f0807))
* AboutPage component imported to AboutView ([ab55300](https://github.com/joeytierney/dbd-vue/commit/ab55300676f4554851ea46654f36f23320aea182))
* home page content and styles ([cea84cc](https://github.com/joeytierney/dbd-vue/commit/cea84ccb5f3d5c3b252db3103a62d6cb923d1786))
* homepage style updates, survivor button imported ([9e7592f](https://github.com/joeytierney/dbd-vue/commit/9e7592f0e367f151f9fdabf974a7c51587aef2a9))
* killer home section style and button import ([7e7e603](https://github.com/joeytierney/dbd-vue/commit/7e7e603f7df793f6f2ab219fe5663063b0621ee2))
* privacy policy page content, style updates, router updates ([52905e1](https://github.com/joeytierney/dbd-vue/commit/52905e10290b1422dbfdb8ac3fd17228b9a84900))
* Sheva and Carlos characters added ([76d9666](https://github.com/joeytierney/dbd-vue/commit/76d966662b1c38d218b2c901ac0382d94d592684))

### [0.1.7](https://github.com/joeytierney/dbd-vue/compare/v0.1.6...v0.1.7) (2022-08-22)


### Features

* footer component ([f5e83ef](https://github.com/joeytierney/dbd-vue/commit/f5e83ef0e240bd5c135a2db06200dc51411db5b0))
* links added to social media icons ([9139b3b](https://github.com/joeytierney/dbd-vue/commit/9139b3b379a4ab4deb6a191276486255a548d7d7))
* social media icons added to footer ([8e38d01](https://github.com/joeytierney/dbd-vue/commit/8e38d019990d864dc3135c34d9e44526be86340d))

### [0.1.6](https://github.com/joeytierney/dbd-vue/compare/v0.1.5...v0.1.6) (2022-08-22)


### Features

* firefox navbar, navbar updates ([3ea94b8](https://github.com/joeytierney/dbd-vue/commit/3ea94b8628bea3cbb6ce260ed1cdd5848096ebc7))
* NavBar component created, DBD logo added and transparent design ([e18592b](https://github.com/joeytierney/dbd-vue/commit/e18592b543dddb89bf5e2d10c608165fdede58cc))
* navbar mobile menu and style ([f41f36c](https://github.com/joeytierney/dbd-vue/commit/f41f36cb33d6915604ee6ea75fb85ea48a3672c0))
* router-links to navbar ([03dee05](https://github.com/joeytierney/dbd-vue/commit/03dee0578e58293e7e9350af50f034412ce36638))
* tailwind config file to include firefox navbar ([cffd1c2](https://github.com/joeytierney/dbd-vue/commit/cffd1c2d111c3b3844354b0c298227e430676456))

### [0.1.5](https://github.com/joeytierney/dbd-vue/compare/v0.1.4...v0.1.5) (2022-08-15)


### Features

* killer and survivor perks added to array ([6d79cb6](https://github.com/joeytierney/dbd-vue/commit/6d79cb6641ee568938dcdca00b982b1fe7281cec))
* killer and survivors from chapter 25 added to arrays ([85e91fe](https://github.com/joeytierney/dbd-vue/commit/85e91fe443facd01fecec7b8b6641634410dc104))
* killer perks imported to view ([255b64b](https://github.com/joeytierney/dbd-vue/commit/255b64b6b640a11ef7429b8ad4a5864df4e29c3b))
* killer real name and styles ([6496966](https://github.com/joeytierney/dbd-vue/commit/6496966b806aae7e078ec90608b3feb103392d5a))
* perk placeholder updated for killer perks ([aaa5bdb](https://github.com/joeytierney/dbd-vue/commit/aaa5bdbb72dc5fedec23e5a8f673e05048c755a1))
* repeat placeholder image 4 times to match 4 perks ([f04966f](https://github.com/joeytierney/dbd-vue/commit/f04966f5d0d9aee660dbb10aa0439b24da71a1ee))


### Bug Fixes


src/utils/killer/KillerArray.js
export const killersArray = [
  {
    name: "The Trapper",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/01-the-trapper.png",
    realName: "Evan MacMillan",
    dlc: "Base Game",
  },
  {
    name: "The Wraith",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/02-the-wraith.png",
    realName: "Philip Ojomo",
    dlc: "Base Game",
  },
  {
    name: "The Hillbilly",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/03-the-hillbilly.png",
    realName: "Max Thompson Jr.",
    dlc: "Base Game",
  },
  {
    name: "The Nurse",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/04-the-nurse.png",
    realName: "Sally Smithson",
    dlc: "CHAPTER 1: The Last Breath",
  },
  {
    name: "The Shape",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/05-the-shape.png",
    realName: "Michael Myers",
    dlc: "CHAPTER 2: HALLOWEEN®",
  },
  {
    name: "The Hag",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/06-the-hag.png",
    realName: "Lisa Sherwood",
    dlc: "CHAPTER 3: Of Flesh and Mud",
  },
  {
    name: "The Doctor",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/07-the-doctor.png",
    realName: "Herman Carter",
    dlc: "CHAPTER 4: Spark of Madness",
  },
  {
    name: "The Huntress",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/08-the-huntress.png",
    realName: "Anna",
    dlc: "CHAPTER 5: A Lullaby for the Dark",
  },
  {
    name: "The Cannibal",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/09-the-cannibal.png",
    realName: "Bubba Sawyer",
    dlc: "CHAPTER 5.5: LEATHERFACE™",
  },
  {
    name: "The Nightmare",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/10-the-nightmare.png",
    realName: "Freddy Krueger",
    dlc: "CHAPTER 6: A Nightmare on Elm Street™",
  },
  {
    name: "The Pig",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/11-the-pig.png",
    realName: "Amanda Young",
    dlc: "CHAPTER 7: SAW™",
  },
  {
    name: "The Clown",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/12-the-clown.png",
    realName: "Jeffrey Hawk",
    dlc: "CHAPTER 8: Curtain Call",
  },
  {
    name: "The Spirit",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/13-the-spirit.png",
    realName: "Rin Yamaoka",
    dlc: "CHAPTER 9: Shattered Bloodline",
  },
  {
    name: "The Legion",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/14-the-legion.png",
    realName: "Frank, Julie, Susie & Joey",
    dlc: "CHAPTER 10: Darkness Among Us",
  },
  {
    name: "The Plague",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/15-the-plague.png",
    realName: "Adiris",
    dlc: "CHAPTER 11: Demise of the Faithful",
  },
  {
    name: "The Ghost Face",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/16-the-ghost-face.png",
    realName: "Danny Johnson",
    dlc: "CHAPTER 12: Ghost Face®",
  },
  {
    name: "The Demogorgon",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/17-the-demogorgon.png",
    realName: "DemoPls",
    dlc: "CHAPTER 13: Stranger Things",
  },
  {
    name: "The Oni",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/18-the-oni.png",
    realName: "Kazan Yamaoka",
    dlc: "CHAPTER 14: Cursed Legacy",
  },
  {
    name: "The Deathslinger",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/19-the-deathslinger.png",
    realName: "Caleb Quinn",
    dlc: "CHAPTER 15: Chains of Hate",
  },
  {
    name: "The Executioner",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/20-the-executioner.png",
    realName: "Pyramid Head",
    dlc: "CHAPTER 16: Silent Hill",
  },
  {
    name: "The Blight",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/21-the-blight.png",
    realName: "Talbot Grimes",
    dlc: "CHAPTER 17: Descend Beyond",
  },
  {
    name: "The Twins",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/22-the-twins.png",
    realName: "Victor & Charlotte Deshayes",
    dlc: "CHAPTER 18: A Binding of Kin",
  },
  {
    name: "The Trickster",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/23-the-trickster.png",
    realName: "Ji-Woon Hak",
    dlc: "CHAPTER 19: All-Kill",
  },
  {
    name: "The Nemesis",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/24-the-nemesis.png",
    realName: "Nemesis T-Type",
    dlc: "CHAPTER 20: Resident Evil™",
  },
  {
    name: "The Cenobite",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/25-the-cenobite.png",
    realName: "Elliot Spencer",
    dlc: "CHAPTER 21: Hellraiser™",
  },
  {
    name: "The Artist",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/26-the-artist.png",
    realName: "Carmina Mora",
    dlc: "CHAPTER 22: Portrait of a Murder",
  },
  {
    name: "The Onryō",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/27-the-onryo.png",
    realName: "Sadako Yamamura",
    dlc: "CHAPTER 23: Sadako Rising",
  },
  {
    name: "The Dredge",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/28-the-dredge.png",
    realName: "Unknown",
    dlc: "CHAPTER 24: Roots of Dread",
  },
  {
    name: "The Mastermind",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/29-the-mastermind.png",
    realName: "Albert Wesker",
    dlc: "CHAPTER 25: Resident Evil™: PROJECT W",
  },
  {
    name: "The Knight",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/30-the-knight.png",
    realName: "Tarhos Kovács",
    dlc: "CHAPTER 26: Forged in Fog",
  },

  // **************************************
  // LEGENDARY AND ULTRA-RARE SKINS SECTION
  // **************************************

  {
    name: "The Krampus",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/01a-the-krampus.png",
    realName: "Holiday Horror",
    dlc: "ULTRA-RARE SKIN FOR: The Trapper",
  },
  {
    name: "The Look-See",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/07a-the-look-see.png",
    realName: "The Crypt TV",
    dlc: "LEGENDARY SKIN FOR: The Doctor",
  },
  {
    name: "The Mordeo",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/08a-the-mordeo.png",
    realName: "The Crypt TV",
    dlc: "LEGENDARY SKIN FOR: The Huntress",
  },
  {
    name: "The Baba Yaga",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/08b-the-baba-yaga.png",
    realName: "Modern Tales",
    dlc: "ULTRA-RARE SKIN FOR: The Huntress",
  },
  {
    name: "The War Hammer Titan",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/13a-the-war-hammer-titan.png",
    realName: "Lara Tybur",
    dlc: "ULTRA-RARE SKIN FOR: The Spirit",
  },
  {
    name: "The Minotaur",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/18a-the-minotaur.png",
    realName: "Greek Legends",
    dlc: "ULTRA-RARE SKIN FOR: The Oni",
  },
  {
    name: "The Armoured Titan",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/18b-the-armoured-titan.png",
    realName: "Reiner Braun",
    dlc: "ULTRA-RARE SKIN FOR: The Oni",
  },
  {
    name: "The Ferryman",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/21a-the-ferryman.png",
    realName: "Greek Legends",
    dlc: "ULTRA-RARE SKIN FOR: The Blight",
  },
  {
    name: "The Chatterer",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/25a-the-chatterer.png",
    realName: "Jim",
    dlc: "LEGENDARY SKIN FOR: The Cenobite",
  },
  {
    name: "The Grim Reaper",
    image:
      "https://raw.githubusercontent.com/joeytierney/dbd-vue/develop/src/assets/character-portraits/killers/14a-the-grim-reaper.png",
    realName: "HUNK",
    dlc: "LEGENDARY SKIN FOR: The Legion",
  },
];
